src/Illuminate/Http/Middleware/SetCacheHeaders.php

```diff
@@ -31,6 +31,18 @@ public function handle($request, Closure $next, $options = [])
         if (isset($options['etag']) && $options['etag'] === true) {
             $options['etag'] = md5($response->getContent());
         }
+        
+        if (isset($options['last_modified']))
+		{
+			if (is_numeric($options['last_modified'])) 
+			{
+                $options['last_modified']=new \DateTime('@'.$options['last_modified']);
```

Comment:
Laravel uses the Carbon package internally which allows mocking of the time for tests.

Mentioning the tests, maybe some would be good.